core/citrus-base/src/main/java/org/citrusframework/validation/script/TemplateBasedScriptBuilder.java

```diff
@@ -64,9 +64,8 @@ public String build() {
         StringBuilder scriptBody = new StringBuilder();
         String importStmt = "import ";
 
-        try {
+        try (BufferedReader reader = new BufferedReader(new StringReader(scriptCode));) {
```

Comment:
```suggestion
        try (BufferedReader reader = new BufferedReader(new StringReader(scriptCode))) {
```

Reply:
Yeah, nice catch. I didn't mean to add them. :)
Removed them in a new commit.